HomeWorks/HW03.Task1/Program.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+
+namespace HW03.Task1
+{
+    class Calculator
+    {
+        public double Add(double firstValue, double secondValue)
+        {
+            return firstValue + secondValue;
+        }
+        public double Subtruct(double firstValue, double secondValue)
+        {
+            return firstValue - secondValue;
+        }
+        public double Multiply(double firstValue, double secondValue)
+        {
+            return firstValue * secondValue;
+        }
+        public double Devide(double firstValue, double secondValue)
+        {
+
+            if (secondValue == 0)
+            {
+                Console.WriteLine("The second value is set to zero, it's impossible to devide by zero");
+                return firstValue / secondValue;
+            }
+            else
+            {
+                return firstValue / secondValue;
+            }
+        }
+        public int ReminderAfterDeviding(int devident, int devider)
+        {
+            return devident % devider;
+        }
+        public double CircleSquare(double Radius)
+        {
+            return 3.14 * Radius * Radius;
+        }
+    }
+    class Program
```

Comment:
the same - add empty line. 
Better to move type `Calculator` as a separate C# file